order.go

```diff
@@ -52,6 +52,12 @@ func (c *Client) CancelOrder(id string) error {
 	return err
 }
 
+func (c *Client) CancelAll() ([]string, error) {
```

Comment:
Can we rename this to CancelAllOrders?

Also can we have this support the optional product_id query parameter? You can do something like with ListOrdersParams and then: func (c *Client) CancelAllOrders(p ...CancelAllOrdersParams) ([]string, error)

Reply:
Absolutely. Would you prefer it to be an "optional" parameter, or two distinct methods; `CancelAllOrders` and `CancelAllOrdersForProduct` ?

Reply:
"optional" parameter